tests/integration/components/lt-head-test.js

```diff
@@ -62,6 +63,35 @@ test('click - sortable column', function(assert) {
   click(sortableHeader);
 });
 
+test('render sort icons', function(assert) {
+  this.set('table', new Table(Columns));
+
+  this.render(hbs`{{lt-head table=table renderInPlace=true iconSortable='fa-sort' iconAscending='fa-sort-asc' iconDescending='fa-sort-desc'}}`);
+
+  let allHeaders = findAll('tr > th ');
+  let sortableHeader = allHeaders[allHeaders.length - 1];
+  let sortIcon = find('.lt-sort-icon', sortableHeader);
+
+  // Sortable case
+  assert.ok(hasClass(sortIcon, 'fa-sort'), 'Sortable icon renders');
```

Comment:
woops :door: :walking_man:  didn't pay close enough attention to the default.
you're right.